common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/Utils.java

```diff
@@ -42,4 +47,27 @@ public static String sanitizeBoolean(String value, DataField.DataTypeEnum dataTy
     }
     return value;
   }
+
+  /**
+   * Check if prediction interval is enabled or not.
+   */
+  public static boolean isPredictionIntervalEnabled() {
+    return isPredictionIntervalEnabledFromEnv() || isPredictionIntervalEnabledFromProperty();
+  }
+
+  private static boolean isPredictionIntervalEnabledFromEnv() {
+    try {
+      return Boolean.getBoolean(PREDICTION_INTERVAL_ENABLE_PROPERTY);
+    } catch (Exception ignored) {
+      return false;
+    }
```

Comment:
The [only documented exception](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Boolean.html#getBoolean(java.lang.String)) this can throw is a SecurityException, which we should arguably not catch here, let alone catch and ignore.

Is there another reason this is consuming the exception?  If not, please consider simply returning `Boolean.getBoolean(...)`.

Reply:
make sense